engine/schema/src/main/java/com/cloud/host/HostVO.java

```diff
@@ -740,6 +743,18 @@ public void setUuid(String uuid) {
         this.uuid = uuid;
     }
 
+    public boolean checkHostServiceOfferingTags(ServiceOffering serviceOffering){
+        if (serviceOffering == null) {
+            return false;
+        }
+        if (Strings.isNullOrEmpty(serviceOffering.getHostTag())) {
```

Comment:
sorry to keep nagging you, but we have a string utils proxy class in place. It has an isEmpty() that does the same check. if for some reason you need this one anyway , please add it there if the functioonality doesn't suffice.

Reply:
@DaanHoogland Is ok, I think I will survive this :D
This method was not my idea, it was a part of the old implementation and I thought it was nice, so I reused this.

I will change it ;)